src/harness/security_testcase.py

```diff
@@ -264,16 +264,20 @@ def assertTlsHandshakeFailureOrHttp403(self, client_cert=None, client_key=None,
         given |client_cert|. If 'None' the default CBSD key file will be used.
       ciphers: optional cipher method
       ssl_method: optional ssl_method
+      is_sas: boolean to determinate next request
     """
     try:
       self.assertTlsHandshakeFailure(client_cert, client_key, ciphers, ssl_method)
     except AssertionError as e:
       try:
-        device_a = json.load(
-          open(os.path.join('testcases', 'testdata', 'device_a.json')))
-        request = {'registrationRequest': [device_a]}
-        response = self._sas.Registration(request, ssl_cert=client_cert,
-                                          ssl_key=client_key)['registrationResponse']
+        if is_sas:
+          self._sas.GetFullActivityDump(client_cert, client_key)
```

Comment:
I think we need to trigger FAD generation first; otherwise, how should a SAS respond to the first FAD request after Reset()?

Please check the end of our meeting notes from our April 10 meeting -- they explain the test dev decision on how to handle this.

Reply:
I haven't seen "Javi's email", so please wait for  Kate to confirm that this is consistent with it.